ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -270,6 +270,7 @@
 {% include 'eos/router-bgp.j2' %}
 {# router igmp configuration #}
 {% include 'eos/router-igmp.j2' %}
+{% include 'eos/router-internet-exit.j2' %}
```

Comment:
1/ please add comment as for the others
2/ have you checked the place in the configuration is correct?

Reply:
1/ Done!
2/ Checked and changed its place.